packages/workspace/src/utils/rules/to-js.ts

```diff
@@ -1,12 +1,25 @@
 import { transpile, JsxEmit, ScriptTarget } from 'typescript';
-import { forEach, Rule, when } from '@angular-devkit/schematics';
+import {
+  forEach,
+  when,
+  chain,
+  CreateFileAction,
+  Rule,
+} from '@angular-devkit/schematics';
 import { normalize } from '@angular-devkit/core';
+import { updateJsonInTree } from '../ast-utils';
 
-export function toJS(): Rule {
+const uniq = <T extends string[]>(value: T) => [...new Set(value)] as T;
+
+const changeFileExtensions: Rule = (tree, context) => {
```

Comment:
Does this change all `.ts` files in the workspace to `.js`?

Reply:
It wasn't supposed to but it probably needs some scoping